plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtPvlanSetupCommandWrapper.java

```diff
@@ -87,7 +88,7 @@ public Answer execute(final PvlanSetupCommand command, final LibvirtComputingRes
             } else if (command.getType() == PvlanSetupCommand.Type.VM) {
                 final String ovsPvlanVmPath = libvirtComputingResource.getOvsPvlanVmPath();
 
-                final Script script = new Script(ovsPvlanVmPath, timeout, s_logger);
+                final Script script = new Script(ovsPvlanVmPath, timeout.getMillis(), s_logger);
```

Comment:
@jburwell yes you are right, we should modify the script class to take duration as timeout instead of long.